camp/camp-base/src/main/java/org/apache/brooklyn/camp/spi/pdp/AssemblyTemplateConstructor.java

```diff
@@ -40,15 +40,22 @@ public AssemblyTemplateConstructor(CampPlatform campPlatform) {
         this.transaction = this.campPlatform.transaction();
     }
     
-    /** records all the templates to the underlying platform */
+    /** records all the templates to the underlying platform 
+     * @deprecated since 0.10.0 use {@link #construct(boolean)} */
     public AssemblyTemplate commit() {
+        return construct(false);
```

Comment:
I see this is now deprecated and not called, but why change the semantics to not actually commit (given that it will now do `transaction.commit()`)? If deprecating, why not leave the semantics as before? Or delete the method entirely?